spring-cloud-gcp-core/pom.xml

```diff
@@ -19,12 +19,12 @@
 			<artifactId>google-cloud-core</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework</groupId>
-			<artifactId>spring-core</artifactId>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
```

Comment:
This doesn't feel right...
`spring-cloud-gcp-core` should not depend on Spring Boot. It should be usable standalone.

Also, since every module depends on `spring-cloud-gcp-core`, Spring Boot would automatically spread to the whole project?

Reply:
I  agree that the dependency on the starter is wrong but I still haven't seen a single reason why this module can't depend on Spring  Boot considering the fact the code is related to a usage of Spring Cloud and the latter requires the former.

Reply:
I don't get it either. I don't see any reason to make this available to apps that don't use Spring Boot.